src/events.h

```diff
@@ -72,6 +74,7 @@ class Events
 		bool eventCreatureOnChangeOutfit(Creature* creature, const Outfit_t& outfit);
 		ReturnValue eventCreatureOnAreaCombat(Creature* creature, Tile* tile, bool aggressive);
 		ReturnValue eventCreatureOnTargetCombat(Creature* creature, Creature* target);
+		void eventCreatureOnHear(Creature* creature, Creature* speaker, const std::string words, SpeakClasses type, Position pos);
```

Comment:
if you arent removing that position, it should be passed by reference and const like words (which isnt)

Reply:
done